doc/events.md

```diff
@@ -90,5 +93,33 @@ cat events.json | origo events put <datasetid> <versionid>
 origo events put my-dataset 1 --file=events.json
 ```
 
-# Consuming events
-TBD
+# Sinks
+
+Event stream sinks (`s3` or `elasticsearch`) can be added using the following command:
+
+```bash
+origo events add-sink <datasetid> <versionid> --sink-type=<sink_type>
```

Comment:
Istedenfor å ha `datasetid` og `version` som separate parametere, så ønsker vi istedenfor å standardisere på å bruke en "dataset URI" slik vi allerede har gjort for f.eks. `origo datasets cp`.

Noen gyldige dataset-URIer:
- `ds:my-dataset/1` - Full spesifikasjon av URI namespace (`ds:`), datasett-id og versjon
- `my-dataset/1` - Uten `ds:` namespace, der det ikke er påkrevd
- `my-dataset` - Kun datasett-id angitt (default version = 1)

Reply:
Se også "CLI Rules/best practices": https://docs.google.com/spreadsheets/d/18XLmhplCUeZI70cw4RnbTFrj2CLqOpMrQyZgzMsak2E/edit#gid=226673526